terraform/team-members-access/_users.tf

```diff
@@ -18,6 +18,7 @@ locals {
     "shepmaster"    = [aws_iam_group.infra_deploy_playground.name, aws_iam_group.infra_team.name],
     "oli-obk"       = [aws_iam_group.infra_deploy_staging_dev_desktop.name],
     "LawnGnome"     = [aws_iam_group.crates_io.name],
+    "Kobzol"        = [aws_iam_group.rustc_perf.name],
```

Comment:
Jakub, is this username fine for you? Do you prefer another one? Or maybe this lowercase? Let me know :)

Reply:
Username is fine like this, thanks.